### PR TITLE
4.4.2 (small fixes)

### PR DESCRIPTION
**Pull request by TBG-FR**
Hello there !

Two small fixes for 4.4.2
- Hover on channel members (linked to #168 
- Switch toggle color not visible

Didn't noticed any other issue related to new version, but don't hesitate to report them !

### CODE REVIEW

**Comment by pbaity** [comment]
Hey @TBG-FR , thanks again! For me this doesn't fix the light-colored hover in the channel members list:

image

Is that what you are referring to?

**Comment by TBG-FR** [comment]
You're right, I forgot to add `body.dark-mode` before the rule, that should be fixed now

File: dark-mode.css [hunk 1]
```diff
@@ -573,6 +573,11 @@ body.dark-mode button.rcx-contextual-message__follow + div.rcx-box--full {
 body.dark-mode .rcx-option__content {
 	color: var(--color-gray-light);
 }
+
+body.dark-mode .rcx-option--focus, .rcx-option:hover {
+    background-color: var(--rc-color-primary-dark);
```
Comment (pbaity) [inline dark-mode.css:578]: @TBG-FR For me this rule still doesn't work, it's not overriding the default rule. We could override the default rule with an `!important` here, but I think I found another solution. Here's the regular rule:
```css
.rcx-option--focus, .rcx-option:hover {
    background: #f7f8fa;
    background: var(--rcx-color-neutral-100,#f7f8fa);
}
```
We could instead overwrite this var in our dark mode vars section:
```css
body.dark-mode {
    /* ... */
    --rcx-color-neutral-100: var(--rc-color-primary-dark);
    /* ... */
}
```
Let me know if this works for you and what you think!

Comment (TBG-FR) [reply dark-mode.css:578]: You're right, looks like I missed something while testing that fix... Yours works perfectly well, I'll fix this right now !

**Comment by TBG-FR** [comment]
> rather

I agree with you, I prefer using the variables too. However, your suggestion doesn't fix #172 (because it uses a custom rule 😞 )
I'll fix these two anyway, commits on the way 😉 Thank you for reviewing that !

**Comment by TBG-FR** [comment]
> @TBG-FR When you have time, see my last comment and let me know what you think. I prefer using the variables rather than overwriting rules when possible. (Just a reminder since issues #172 and #173 came in, and this should fix those.)

I agree with you, I prefer using the variables too. However, your suggestion doesn't fix https://github.com/pbaity/rocketchat-dark-mode/issues/172 (because it uses a custom rule 😞 )
I'll fix these two anyway, commits on the way 😉 Thank you for reviewing that !

**Review by pbaity — APPROVED** [review]
Looks great! Thanks. For some reason I can't confirm that #172 is fixed because I don't have a message info option...? But I don't want to hold up this PR any longer, if anyone finds the issue is not fixed, we'll fix later. 

Excellent work @TBG-FR !

**Pull request merged**